hexa/plugins/connector_iaso/views.py

```diff
@@ -0,0 +1,76 @@
+import uuid
+from logging import getLogger
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import get_object_or_404, render
+from django.utils.translation import gettext_lazy as _
+
+from .datacards import FormCard, IASOCard
+from .datagrids import ObjectGrid
+from .models import IASOAccount
+
+logger = getLogger(__name__)
+
+
+def datasource_index(request: HttpRequest, datasource_id: uuid.UUID) -> HttpResponse:
+    iaso_account = get_object_or_404(
+        IASOAccount.objects.filter_for_user(request.user),
+        pk=datasource_id,
+    )
+
+    iaso_card = IASOCard(iaso_account, request=request)
+
+    breadcrumbs = [
+        (_("Catalog"), "catalog:index"),
+        (iaso_account.display_name, "connector_iaso:datasource_index", datasource_id),
+    ]
+
+    object_grid = ObjectGrid(
+        iaso_account.iasoform_set.prefetch_indexes().select_related("iaso_account"),
+        parent_model=iaso_account,
+        prefix="",
+        per_page=20,
+        page=int(request.GET.get("page", "1")),
+        request=request,
+    )
+
+    return render(
+        request,
+        "connector_iaso/iaso_index.html",
+        {
+            "datasource": iaso_account,
+            "breadcrumbs": breadcrumbs,
+            "iaso_card": iaso_card,
+            "object_grid": object_grid,
+        },
+    )
+
+
+def iasoform_detail(
```

Comment:
`form_detail` is fine - we haven't prefixed our other views with the plugin name until now